wrappers/ios/vcx/VcxAPI.m

```diff
@@ -835,22 +835,19 @@ - (int)vcxRevocationRegistryRelease:(NSNumber *)revRegHandle {
 
 - (void)vcxCredentialDefinitionCreateV2:(NSString *)sourceId
                                schemaId:(NSString *)schemaId
-                              issuerDid:(NSString *)issuerDid
```

Comment:
just double checking - is there any updates needed to the android/java wrapper APIs?

Reply:
as far as I am aware, all places in wrappers has been updated respectively